Tools/setup/ubuntu.sh

```diff
@@ -263,15 +317,50 @@ if [[ $INSTALL_SIM == "true" ]]; then
 		pkg-config \
 		protobuf-compiler \
 		;
+fi
```

Comment:
The installation of `gstreamer` and the other packages is already done in `install_gazebo_common`